### DISCUSSION TITLE
Load different agents as my wish

### DISCUSSION THREAD

**Discussion by Into-sight** [waymo-research/waymax#83] [Q&A]
As far as I know, the demo is controlled through the index of the agent, but is there any way to control the agent through other conditions, such as its tpye?

**Comment by samuela** [comment]
Yes, you can specify arbitrary policies for each of the agents. See eg https://github.com/waymo-research/waymax/blob/main/docs/notebooks/multi_actors_demo.ipynb for some example code!
